.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,55 @@
+name: Pull request comment
+
+on:
+  pull_request_target:
+    paths:
+      - 'icons/*.svg'
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  Explore-GitHub-Actions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: refs/pull/${{ github.event.pull_request.number }}/merge
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v35
+        with:
+          files: icons/*.svg
+      - name: Generate comment
+        id: generate-comment
+        run: |
+          delimiter="$(openssl rand -hex 8)"
+          echo "body<<$delimiter" >> $GITHUB_OUTPUT
+          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            cat "$file" | # get file content
+            tr '\n' ' ' | # remove line breaks
+            sed -e 's/<svg[^>]*>/<svg>/g' | # remove attributes from svg element
+            base64 -w 0 | # encode svg
+            sed "s|.*|<picture><source media=\"(prefers-color-scheme: dark)\" srcset=\"https://lucide.dev/api/gh-icon/&-dark.svg\"/><source media=\"(prefers-color-scheme: light)\" srcset=\"https://lucide.dev/api/gh-icon/&.svg\"/><img width=\"48%\" title=\"$file\" alt=\"$file\" src=\"https://lucide.dev/api/gh-icon/&.svg\"/></picture> |"
```

Comment:
Why does this have to be 48%?

Reply:
Both work, but the bottom one looks better with the spacing between the images.
```html
<img w="50%"/><img w=50%/>
<img w="48%"/> <img w=48%/>
```

Reply:
I see. I wonder if some degree of responsivity could be achieved, but can't really see a very obvious solution. 🤔

Reply:
This is kind of hacky, but `width="400"` currently works kind of OK (on desktop it's two icons in a row, on mobile it's 1 icon at full width).